samples/client-sdk/connectivity-tests/src/call/controls.js

```diff
@@ -0,0 +1,76 @@
+import { hideTestResults, removeAllParticipantEles } from './dom.js';
+
+/**
+ * Configures the test button onclick handler
+ * @param {()} handler
+ */
+export function setupTestBtn(handler) {
+  const testBtn = getTestBtn();
+  testBtn.onclick = (ev) => {
+    ev.preventDefault();
```

Comment:
General feedback but I find this level of abstraction for the buttons a bit hard to read when the functions are only used once. When it's refactored this early, it suggests it's used in multiple places. Personally, I would just add the onclick event where this function is currently being used in index.js. Same for the other buttons.

Reply:
My reasoning was to keep as much of the DOM stuff as possible in a separate file so that the `index.js` file could focus purely on the test methods being shown and the Daily setup itself. Especially since I have a feeling this sample may be expanded at some point to also show deeper usage of these test methods in-call (which would mean more DOM controls and such being added). And if that were the case I'd probably end up moving a bunch of the controls into this separate file structure down the line to avoid cramming more into one file, creating a bigger diff (_or_ requiring an additional preceding PR to refactor before adding the new functionality)

So for now I think I'll tentatively keep this as is (may change my mind later :D)

Reply:
All good, up to you!